modules/ROOT/pages/ocis_release_notes.adoc

```diff
@@ -417,7 +417,10 @@ The event bus used in many services is now configurable with a set of environmen
 The following services have been added:
 
 * `auth-service`: +
-The Infinite Scale auth-service is used to authenticate service accounts. Compared to normal accounts, service accounts are Infinite Scale internal and not available to ordinary users like via LDAP. https://github.com/owncloud/ocis/pull/6427[#6427]
+The Infinite Scale auth-service is used to authenticate service accounts. Compared to normal accounts, service accounts are Infinite Scale internal and not available to ordinary users like via LDAP. https://github.com/owncloud/ocis/pull/6427[#6427] +
+--
+WARNING: Service accounts are a breaking change for instances which are upgrading from Infinite Scale 4.0.0 and before. Please make sure to carefully read the [upgrade instructions].
```

Comment:
@mmattel can you add the correct :xref ?

Reply:
done